hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneClientFactory.java

```diff
@@ -136,6 +136,31 @@ public static OzoneClient getRpcClient(String omHost, Integer omRpcPort,
     return getRpcClient(config);
   }
 
+  /**
+   * Returns an OzoneClient which will use RPC protocol.
+   *
+   * @param omServiceId
+   *        Service ID of OzoneManager HA cluster.
+   *
+   * @param config
+   *        Configuration to be used for OzoneClient creation
+   *
+   * @return OzoneClient
+   *
+   * @throws IOException
+   */
+  public static OzoneClient getRpcClient(String omServiceId,
+      Configuration config)
+      throws IOException {
+    Preconditions.checkNotNull(omServiceId);
+    Preconditions.checkNotNull(config);
+    // Override ozone.om.address just in case it is used later.
+    // Because if this is not overridden, the (incorrect) value from xml
+    // will be used?
+    config.set(OZONE_OM_ADDRESS_KEY, omServiceId);
```

Comment:
I didn't understand why this is required. Let's discuss tomorrow.

Reply:
As discussed, I will remove this one. Thanks for pointing out!